Scripts/demo

```diff
@@ -41,7 +41,7 @@ class Runner
   def cleanup
     puts 'Cleaning up server'
     system('pkill ngrok')
-    system('pkill \'StytchDemo \(Server\)\'')
```

Comment:
Since we killed `make demo`, I think we can delete this file

Reply:
fixed